### PR TITLE
New packages : liborc , libspice-server , libspice-protocol

### PR DESCRIPTION
**Pull request by mjuned47**

### CODE REVIEW

File: packages/liborc/build.sh [hunk 1]
```diff
@@ -0,0 +1,16 @@
+TERMUX_PKG_DESCRIPTION="Library of Optimized Inner Loops Runtime Compiler"
```
Comment (xeffyr) [inline packages/liborc/build.sh:1]: Where is the homepage?

File: packages/liborc/build.sh [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+TERMUX_PKG_LICENSE="LGPL-2.1"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION=0.4.32
+TERMUX_PKG_SRCURL=https://github.com/GStreamer/orc/archive/refs/tags/${TERMUX_PKG_VERSION}.tar.gz
+TERMUX_PKG_SHA256=6a7349d2ab4a73476cd4de36212e8c3c6524998081aaa04cf3a891ef792dd50f
+TERMUX_PKG_BREAKS="liborc"
+TERMUX_PKG_REPLACES="liborc"
```
Comment (xeffyr) [inline packages/liborc/build.sh:8]: Could you elaborate why package should break/replace itself? Are you sure you correctly understand what these fields do?

File: packages/libspice-protocol/build.sh [hunk 3]
```diff
@@ -0,0 +1,8 @@
+TERMUX_PKG_DESCRIPTION="SPICE protocol headers"
+TERMUX_PKG_LICENSE="LGPL-2.1"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION=0.14.3
+TERMUX_PKG_SRCURL=https://www.spice-space.org/download/releases/spice-protocol-${TERMUX_PKG_VERSION}.tar.xz
+TERMUX_PKG_SHA256=f986e5bc2a1598532c4897f889afb0df9257ac21c160c083703ae7c8de99487a
+TERMUX_PKG_BREAKS="libspice-protocol"
+TERMUX_PKG_REPLACES="libspice-protocol"
```
Comment (xeffyr) [inline packages/libspice-protocol/build.sh:8]: Same issues as liborc.

**Review by xeffyr — CHANGES_REQUESTED** [review]

File: packages/liborc/build.sh (hunk 1 above)
Comment (mjuned47) [reply packages/liborc/build.sh:1]: Sorry i will add for both liborc and libspice-protocol.
Also i miss understood pkg_breaks and pkg_replaces

**Comment by mjuned47** [comment]
When will this request accepted?
I have opened pull request for the first time, So i might make some mistakes

File: packages/libspice-protocol/build.sh [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+TERMUX_PKG_DESCRIPTION="SPICE protocol headers"
+TERMUX_PKG_LICENSE="LGPL-2.1"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION=0.14.3
+TERMUX_PKG_SRCURL=https://www.spice-space.org/download/releases/spice-protocol-${TERMUX_PKG_VERSION}.tar.xz
+TERMUX_PKG_SHA256=f986e5bc2a1598532c4897f889afb0df9257ac21c160c083703ae7c8de99487a
+
+termux_step_post_make_install() {
+mv "${TERMUX_PREFIX}"/share/pkgconfig/spice-protocol.pc \
+    "${TERMUX_PREFIX}"/lib/pkgconfig
```
Comment (Grimler91) [inline packages/libspice-protocol/build.sh:11]: Please tab-indent this function

Comment (mjuned47) [reply packages/libspice-protocol/build.sh]: Done

Comment (mjuned47) [reply packages/libspice-protocol/build.sh]: I think everything is fine now

**Review by manfredwedderin — APPROVED** [review]

**Comment by mjuned47** [comment]
I guess i fixed everything now , expecting to be allow merging

**Pull request merged**